plugin/src/main/java/com/denizenscript/denizen/events/player/PlayerLoginScriptEvent.java

```diff
@@ -28,7 +28,8 @@ public class PlayerLoginScriptEvent extends BukkitScriptEvent implements Listene
     // @Warning Generally avoid this event. This is not a way to get a 'first join' event. This is an internal technical event, with specific uses (eg custom whitelisting).
     //
     // @Context
-    // <context.hostname> returns an ElementTag of the player's hostname.
+    // <context.hostname> returns an ElementTag of the server address that the player used to connect to the server.

```

Comment:
You can't just casually swap a pre-existing context like this, that will break any existing scripts.

Reply:
Fixed.